extension/content.js

```diff
@@ -7,23 +7,14 @@ const isMacOs = navigator.userAgentData
 function tryFixWatchLaterVideoUrl(element) {
   if (element.tagName !== "A") {
     while (element) {
-      if (
-        element.id === "content" &&
-        element.parentElement?.tagName === "YTD-PLAYLIST-VIDEO-RENDERER"
-      ) {
+      if (element?.tagName === "A") {
```

Comment:
```suggestion
      if (element.tagName === "A") {
```
I think we can drop the safe-navigation operator here since we're making sure `element` is not `null` on line 9.